tokio-util/src/udp/frame.rs

```diff
@@ -93,7 +100,11 @@ impl<C: Decoder + Unpin> Stream for UdpFramed<C> {
     }
 }
 
-impl<I, C: Encoder<I> + Unpin> Sink<(I, SocketAddr)> for UdpFramed<C> {
+impl<I, C, T> Sink<(I, SocketAddr)> for UdpFramed<C, T>
+where
+    T: Borrow<UdpSocket> + Unpin,
+    C: Encoder<I> + Unpin,
```

Comment:
Again, you don't need `Unpin` here.
```suggestion
    T: Borrow<UdpSocket>,
    C: Encoder<I>,
```

Reply:
I've added these and the above changes manually, I just want to call out that the previous bound was `C: Encoder<I> + Unpin` so we are removing an existing `Unpin` bound, same with `impl<C: Decoder + Unpin>`

Reply:
Okay, well, there should not have been any `Unpin` bounds in this file in the first place.